README.md

```diff
@@ -1,98 +1,119 @@
+
+There are cases where you do not want to show the smart app banner on all Android and/or all iOS devices. For example:
+* your app is availabe only for some Android/iOS versions
+* your app is only availabe on iPhone, but not iPad
+* your app is a web app which also shows this website, but of course should not show the smart app banner.
+
+In this case you can define a regular expression, which matches all user agent strings that should be excluded. Just add another `meta` tag like the following:
+```html
+<meta name="smartbanner:exclude-user-agent-regex" content="^.*My Example Webapp$">
+```
+This regular expression would match any user agent string, that ends with *My Example Webapp*.
+
+### Add a separate links specific to some device/platform
+
+There are cases when there are separate applications meant for phone and tablet of the same OS. For this purpose you can use the OS related metatag with platform-device suffix, e.g.
+
+#### iOS - iPhone app link can be set with the following metatag:
+```html
+  <meta name="smartbanner:button-url-apple-iphone" content="https://itunes.apple.com/iphonelink">
+```
+
+#### iOS - iPad app link can be set with the following metatag:
+```html
+    <meta name="smartbanner:button-url-apple-ipad" content="https://itunes.apple.com/ipadlink">
+```
+
+**Note:** Is is important that if 'global' links, like `<meta name="smartbanner:button-url-apple"/>` or `<meta name="smartbanner:button-url-google"/>` will be set, device related app links will be omitted (-phone; -tablet).
+
+### Show the smartbanner for certain User Agents
+
+In addition to blacklisting certain user agents using the regex explained in the previous section, you can also whitelist certain user agents:
+```html
+<meta name="smartbanner:include-user-agent-regex" content="iPhone 7">
+```
+
+**Note:** You can define `enabled-platforms`, `exclude-user-agent-regex` and `include-user-agent-regex` at the same time. `enabled-platforms` has the lowest priority, `exclude-user-agent-regex` the highest priority.
+
+### Disable Positioning
+
+If you want to position smart app banner yourself (e.g. in CSS), you can disable built-in positioning with following option:
+```html
+<meta name="smartbanner:disable-positioning" content="true">
+```
+### Hide the smartbanner completely
+
+If you want to prevent smartbanner rendering in some html pages, you can add optional `meta` tag:
+
+```html
+<meta name="smartbanner:enabled-platforms" content="none">
+```
+
+## Contributing
+
+See [CONTRIBUTING.md](CONTRIBUTING.md).
+
+## Sponsors
+
+Cross-browser testing across all mobile platforms is powered by
+
+<a href="https://www.browserstack.com" title="BrowserStack"><img src="https://rawgithub.com/ain/smartbanner.js/master/browserstack.svg" height="32px" alt="BrowserStack"></a>
+
+## Licence
+
+Copyright © 2016, 2017 Ain Tohvri, contributors. Licenced under [GPL-3](https://raw.githubusercontent.com/ain/smartbanner.js/master/LICENSE).

```

Comment:
Diff unnecessarily too big. Please make sure your code is documented in Advanced part, the diff should only be in that section.